src/NServiceBus.Core/Sagas/CustomFinderAdapter.cs

```diff
@@ -1,8 +1,7 @@
-namespace NServiceBus.Sagas
+namespace NServiceBus.Saga
```

Comment:
@andreasohlund both `Saga` and `Saga<T>` are already in `NServiceBus.Saga` namespace, so I did not change that.
All I am doing here is to standardising on a single namespace for the the saga stuff.

Makes sense ?